packages/babel-runtime/helpers/toArray/index.js

```diff
@@ -0,0 +1,14 @@
+var arrayWithHoles = require("@babel/runtime/helpers/arrayWithHoles");
+
+var iterableToArray = require("@babel/runtime/helpers/iterableToArray");
+
+var unsupportedIterableToArray = require("@babel/runtime/helpers/unsupportedIterableToArray");
+
+var nonIterableRest = require("@babel/runtime/helpers/nonIterableRest");
+
+function _toArray(arr) {
+  return arrayWithHoles(arr) || iterableToArray(arr) || unsupportedIterableToArray(arr) || nonIterableRest();
+}
+
+module.exports = _toArray;
+module.exports["default"] = module.exports;
```

Comment:
Are we sure that the sole existence of `.default` is enough here? It seems that it might be because of the same thing being assigned both to `module.exports` and `module.exports.default`.

When consuming ESM file webpack 4 would load this file directly (as it would just fallback to `/index.js` because it's not aware of `pkg.json#exports`) and if it's using the exact same logic as Babel:
```js
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```
this would work even though `__esModule` is not present here 🤔 I'm wondering though if it still wouldn't be worth adding `__esModule` as maybe some other tools treat this different somehow? Although not sure how that would look like (such a tool's interop logic). Just thinking out loud.